src/main/scala/scala/tools/refactoring/implementations/ClassDefOrganizeImports.scala

```diff
@@ -0,0 +1,8 @@
+package scala.tools.refactoring
+package implementations
+
+trait ClassDefOrganizeImports { self: OrganizeImports =>
```

Comment:
See my comment above: Why a separate trait, instead of keeping this private in `OrganizeImports`?